workbench-core/example/infrastructure/package.json

```diff
@@ -98,8 +98,10 @@
     "eslint": "^8.7.0",
     "eslint-plugin-security": "^1.4.0",
     "express": "^4.17.3",
+    "fast-check": "^2.24.0",
     "istanbul-badges-readme": "1.8.1",
     "jest": "^27.5.1",
+    "jest-fast-check": "^1.0.2",
```

Comment:
are you actually using these new imports?

Reply:
using to randomize objects